play-app/src/Context/ProductCont.jsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
```

Comment:
Suggestion: Above React version 17, importing React from "react" is not necessary.

Reply:
thanks,will keep in mind